lib/runtime/wasmer/instance.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/ChainSafe/gossamer/lib/crypto"
 
 	wasm "github.com/wasmerio/go-ext-wasm/wasmer"
+
+	"github.com/klauspost/compress/zstd"
```

Comment:
Please add `github.com/klauspost/compress v1.12.3` to the list of require dependencies in `go.mod`